src/client/utils/validation.js

```diff
@@ -11,6 +11,11 @@ validator.addFormat('address', {
   validate: isAddress,
 })
 
+export const validateDAOAddress = validator.compile({
+  type: 'string',
+  format: 'address',
```

Comment:
Does this work? Does that actually parse strings and expect an ethereum address?

Reply:
It works, because:

validator.addFormat('address', {
  type: 'string',
  validate: isAddress,
})

and isAddress is the one from web3.